ssg/build_yaml.py

```diff
@@ -1128,7 +1128,7 @@ def to_xml_element(self, env_yaml=None):
         regex = (r'(package_.*_(installed|removed))|' +
                  r'(service_.*_(enabled|disabled))|' +
                  r'install_smartcard_packages$')
-        priority_order = ["installed", "install_smartcard_packages", "removed",
+        priority_order = ["enable_authselect", "installed", "install_smartcard_packages", "removed",
```

Comment:
Is this ordering correct? Do we really want it before `installed` rules?

Reply:
The priority ordering has effect within the group. Could it happen that `authselect` itself is not installed? 
This seems fine as the other package installed rule is `package_pam_pwquality_installed`.
https://github.com/ComplianceAsCode/content/tree/7ad2c970043370def2787a8ed773486384543821/linux_os/guide/system/accounts/accounts-pam

Reply:
The group `accounts-restrictions` also has rules remediating via `authselect`.

So I think we should guarantee that group `accounts-pam` is placed before group `accounts-restriction`  in https://github.com/ComplianceAsCode/content/blob/7ad2c970043370def2787a8ed773486384543821/ssg/build_yaml.py#L1159

Or we move the rule `authselect_enabled` one group up, into the `accounts` group.